tasks/build.py

```diff
@@ -794,7 +794,7 @@ def request_bot_build_issue_comments(repo_name, pr_number):
                 first_line = comment['body'].split('\n')[0]
                 arch_map = get_architecture_targets(cfg)
                 for arch in arch_map.keys():
-                    target_arch = '/'.join(arch.split('/')[1:])
+                    target_arch = '/'.join(arch.split('/')[-1:])
```

Comment:
This might be a tiny bit simpler, right?
```suggestion
                    target_arch = '/'.join(arch.split('/')[-1])
```